src/views/Incorporation/IncorporationAgreement.vue

```diff
@@ -81,17 +81,30 @@
             </div>
           </template>
 
-          <div class="help-div">
-            <h3>Retain the signed Incorporation Agreement and {{ entityDescription }} Articles</h3>
-            <ul>
-              <li>
-                The company is required to keep signed copies of the Incorporation Agreement and
-                Articles in the company’s record book. For a complete list of records, a company
-                is required to keep please see <a :href="section42Url" target="_blank">section 42</a>
-                of the Business Corporations Act.
-              </li>
-            </ul>
-          </div>
+          <template>
+            <div class="help-div" v-if="isTypeBcomp">
```

Comment:
This isn't broken, but the purpose of the template element here is to have the v-if on it and facilitate code folding.

Reply:
@severinbeauvais, if I got you right. Would this be correct way to go?
![image](https://user-images.githubusercontent.com/116035339/206311070-7ef9f8dd-8b72-48b5-adbc-a396f40d0044.png)

Reply:
Yes!

And actually, the template contains a single child, so you could get rid of the template elements and move the [conditional directives](https://vuejs.org/guide/essentials/conditional.html) to the divs instead.

Reply:
Template blocks make more sense when they contain a bunch of children :)

Reply:
> Yes!
> 
> And actually, in this scenario, the template contains a single child, so you could get rid of the template elements and move the [conditional directives](https://vuejs.org/guide/essentials/conditional.html) to the divs instead.

Got it! No template elements and just 2 divs with the conditionals.